.changeset/8s-improve-babel-config.md

```diff
@@ -0,0 +1,20 @@
+---
+issue: https://github.com/pmedianetwork/webpack-config/issues/8
+label: Enhancements
+---
```

Comment:
> How to mark this as a breaking release?

@reakaleek I think we just need todo `type: major` instead of the `label`, right?

Reply:
yes `type: major` will do


You only need a `label`  If you can't provide a type (e.g. documentation changes, or build process changes)
and for "normal" situation you need to choose a `type`  whereas `type` is one of `major`, `minor` or `patch`

Reply:
btw... from my understanding 0.x.x releases are always considered as unstable and breaking. I would only bump to 1.0.0.. if you think this is 1.0.0 worthy...

Reply:
Ok, thanks for the info. Since we're still in `0.x` series, I'll change to `type: minor` to follow SemVer. Eventually we should do 1.0 and do breaking changes with `type: major`.

Reply:
I've just reread semver 2.0.. and it feels like this should be 1.0.0 already :D


>How should I deal with revisions in the 0.y.z initial development phase?
The simplest thing to do is start your initial development release at 0.1.0 and then increment the minor version for each subsequent release.
>
>How do I know when to release 1.0.0?
If your software is being used in production, it should probably already be 1.0.0. If you have a stable API on which users have come to depend, you should be 1.0.0. If you’re worrying a lot about backwards compatibility, you should probably already be 1.0.0.

Reply:
Technically we could call this one as 1.0.0. 👍

Reply:
Ok, I marked this one with `type: major`. It's going to be 1.0.

Before merging, it might be a good idea to look into testing the change against a small local setup so we have means to compare output without having to test against the full projects.

Reply:
can we set this up as some kind of integration test? in a subfolder?

Reply:
probably needs some symlinking to make it work hmmmmfff

Reply:
Yeah, that's what I was thinking. I would do a directory with a small webpack configuration that's running using the code. Technically we could treat the output as snapshots and make sure the build files get updates so they can be reviewed on each PR or so.

Reply:
Do want to do this in this PR or a follow up?

Reply:
I would do a quick setup here so we have a good baseline to compare against. It's not much work to add.

Reply:
Okay nice.. let me know if you need help with the pipeline

Reply:
Sure. 👍